src/hooks/internal/useField.js

```diff
@@ -12,7 +17,17 @@ function useField({ name, disabled, optional, validate, data }) {
     registerField,
     unregisterField
   } = useForm();
+
+  if (typeof state.values === "undefined") {
+    throw new Error(ERROR_STRINGS.FIELD.NO_INITIAL_VALUE);
```

Comment:
I find this abstraction unnecessary. It's harder to read the code this way. Can we just have the error message in plain text here?

Reply:
Yeah I agree, I was originally thinking that some error messages might get reused but I think it's better to make them specific to each use case anyway.